configbp/config.go

```diff
@@ -0,0 +1,67 @@
+// Package configbp parses configurations per the baseplate spec.
+package configbp
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/reddit/baseplate.go/log"
+	"gopkg.in/yaml.v2"
```

Comment:
style nit, this should be:

```suggestion
	"gopkg.in/yaml.v2"

	"github.com/reddit/baseplate.go/log"
```

Reply:
Yep.  In reconciling the editor config I might've accidentally nuked my settings ><